src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -50,14 +52,20 @@ def _check_conditional_args(
             parser.error(
                 "The --endpoint option is required when using the 'openai' service-kind."
             )
-        if args.endpoint == "v1/chat/completions":
-            args.output_format = OutputFormat.OPENAI_CHAT_COMPLETIONS
-        elif args.endpoint == "v1/completions":
-            args.output_format = OutputFormat.OPENAI_COMPLETIONS
+        else:
+            if args.endpoint == "chat":
+                args.output_format = OutputFormat.OPENAI_CHAT_COMPLETIONS
+            elif args.endpoint == "completions":
+                args.output_format = OutputFormat.OPENAI_COMPLETIONS
+            if args.port:
```

Comment:
should this be elif?
```suggestion
            elif args.port:
```
if not, I'm a bit confused by this if-else block. If the user provides `--port`, then this block overwrites `args.endpoint`. How do we determine if it's chat or completions? Are we expecting users to provide `--endpoint` as well?

Reply:
you are right, updated.